aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +59,19 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename F, typename ...Args>
+Tensor& unary_inplace_op_(F op_, const Tensor& self, Tensor& result, Args&&... args) {
```

Comment:
The naming of the function is misleading, it's not an in-place function if values are taken from `self` and saved to `result`. It's an "out" variant.